_sass/api_documentation.scss

```diff
@@ -230,7 +230,7 @@ bulk-tag {
 .exclamation-icon {
   background: var(--dc-red-lite);
   color: white;
-  font-size: 1em;
+  font-size: 0.8em;
```

Comment:
Ah, there was also this issue with the "see also" icon. Worth extracting out some common css rules between the two.